brigade-worker/src/k8s.ts

```diff
@@ -769,7 +769,9 @@ export class JobRunner implements jobs.JobRunner {
         });
         const req = request(requestOptions, (error, response, body) => {
           if (error) {
-            this.logger.error(error.body.message);
+            if (error.body.message) {
```

Comment:
```suggestion
            if (error.body) {
```

@aidarbek Thanks for the PR!

I think you'd rather check for `error.body` here as the error indicates that `error.body` can be undefined:

```
/home/src/dist/k8s.js:518
                this.logger.error(error.body.message);
                                             ^

  TypeError: Cannot read property 'message' of undefined            
```